students/esokada/lesson03/mailroom.py

```diff
@@ -0,0 +1,75 @@
+#imports
+
+import sys
+
+from statistics import mean
+
+from operator import itemgetter
+
+# data structure
+
+donors = [["Usagi", [12.5, 11.17, 130.1]], ["Ami", [17, 230, 2115]],
+["Rei", [510.1, 50]], ["Makoto", [324, 22.7]], ["Minako", [310]]]
+
+
+#functions
+
+def send_thankyou():
+    donorname = input("Please input a name:\n")
+    if donorname == "cancel":
+        return
+    if donorname == "list":
+        for i in donors:
+            print(i[0])
+        donorname = input("Please input a name:\n")
+        if donorname == "cancel":
+            return
```

Comment:
lines 24 to 26 should be de-dented - no need for these to only present on "list" selection as user may not start with that.

also mahy want to use while True loop at the beginning and loop until user selects "list" or enters a name and exit when user types cancel or "C"